README.md

```diff
@@ -180,7 +180,7 @@ Run `ng e2e` to execute the end-to-end tests via [Protractor](http://www.protrac
 
 ## Running pure playwright end-to-end tests
 
-Run `HEAD=true TEST_URL=<testurl> yarn test:fullfunctional2` to execute the pure playwright end-to-end tests via [Playwright](https://playwright.dev/).
+Run `HEAD=true CASEAPIBASEURL=https://manage-case.aat.platform.hmcts.net/cases yarn test:fullfunctional2` to execute the pure playwright end-to-end tests on aat via [Playwright](https://playwright.dev/).
```

Comment:
It's very minor, but it annoys me slightly that the env variable is called CASEAPIBASEURL, since there isn't really an API on that URL. Could we call it APP_URL or something? Doesn't really matter if it's a nuisance

Reply:
Done